python/mlcroissant/mlcroissant/_src/core/constants.py

```diff
@@ -27,6 +27,7 @@ def ML_COMMONS(ctx) -> rdflib.Namespace:
         return ML_COMMONS_V_1_0
 
 
+ML_COMMONS_ARRAY_SHAPE = lambda ctx: ML_COMMONS(ctx).arrayShape
```

Comment:
Should we write a test that the verifier fails if `isArray` is specified with no `arrayShape`? Or is this legit in the specs?

If there's an implicit behaviour that arrayShape defaults to (-1,), we could probably document this

Reply:
After the discussion with Omar:

- `isArray` without `arrayShape` is valid, and in that case `arrayShape` defaults to (-1,). The documentation should specify this behaviour.
- `arrayShape` without `isArray` should make the verifier fail.

Reply:
Godo point, done. I'll update the documentation in a follow-up PR, for now I just updated the docstring.